### PR TITLE
Update to 3.4.4

### PR DESCRIPTION
**Pull request by manics**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

---

This is a copy of @dhirschfeld's https://github.com/conda-forge/podman-feedstock/pull/13 (https://github.com/conda-forge/podman-feedstock/pull/13/commits/4fc896ca3a07f91272d0d3397a3bd92013624853) with these additions:
- Podman 3.4.4
- Modified the `go-licenses` path
- Updated the tests to take account of the changed output from `podman ... info`

I've managed to get this building in a container `quay.io/condaforge/linux-anvil-cos7-x86_64:latest` with command `conda build -c conda-forge -m .ci_support/linux_64_.yaml recipe/`, however this still fails at the testing stage:
- [x] Podman requires conmon >= 2.0.30 (I think), the [conda-forge version 2.0.21 is too old](https://anaconda.org/conda-forge/conmon/)

### CODE REVIEW

**Comment by manics** [comment]
conmon 2.0.31 PR here: https://github.com/conda-forge/conmon-feedstock/pull/10

File: recipe/meta.yaml [hunk 1]
```diff
@@ -94,3 +96,4 @@ about:
 extra:
   recipe-maintainers:
     - mbargull
+    - dhirschfeld
```
Comment (bollwyvl) [inline recipe/meta.yaml:99]: @manics if you're willing to keep this going, adding yourself as a maintainer might help push this over the line (well, that and https://github.com/conda-forge/conmon-feedstock/pull/10 and CDN and a :heavy_check_mark: !)

**Comment by dhirschfeld** [comment]
@conda-forge-admin, please rerender

**Comment by dhirschfeld** [comment]
The `--help` and `--version` tests seemed to work but something else seemed to go wrong:
```
time="2022-01-18T22:54:25Z" level=debug msg="Using OCI runtime \"$PREFIX/bin/runc\""
time="2022-01-18T22:54:25Z" level=info msg="Found CNI network podman (type=bridge) at $PREFIX/etc/cni/net.d/87-podman.conflist"
time="2022-01-18T22:54:25Z" level=debug msg="Default CNI network name podman is unchangeable"
time="2022-01-18T22:54:25Z" level=warning msg="\"/\" is not a shared mount, this could cause issues or missing mounts with rootless containers"
cannot clone: Operation not permitted
Error: cannot re-exec process
Tests failed for podman-3.4.4-h9f03be5_0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
```

**Comment by manics** [comment]
The build is run in docker, and I think we've hit https://github.com/containers/podman/issues/10802

**Comment by bollwyvl** [comment]
Maybe also add :

azure: {store_build_artifacts: true}

So the built artifacts can be tested manually?

https://github.com/conda-forge/jupyterlab-feedstock/blob/master/conda-forge.yml#L6

**Comment by manics** [comment]
The build script can read additional options set in an env-var
https://github.com/conda-forge/podman-feedstock/blob/1f2c62fbe3b4263da8d5f7aa75011bef2ab49c42/.scripts/run_docker_build.sh#L67-L68

Adding `--cap-add SYS_ADMIN` might work? I'll test locally when I get time.

**Comment by manics** [comment]
@bollwyvl thanks for the suggestion, I've added it.

@conda-forge-admin, please rerender

**Comment by dhirschfeld** [comment]
> *Adding `--cap-add SYS_ADMIN` might work? I'll test locally when I get time*

Maybe? Sounds like it might not be an option though :/

image

**Comment by dhirschfeld** [comment]
I'd be ok with not running the test. If we can get it to work great, but if not, we'll pick up any breakage if we're using it regularly and can publish a new build if required. 

image

**Comment by manics** [comment]
`--cap-add` doesn't exist in `docker build`, but it does exist in `docker run` (which is used to build this podman package).

**Comment by manics** [comment]
@conda-forge-admin, please rerender

**Comment by manics** [comment]
Green!

**Review by dhirschfeld — APPROVED** [review]

**Comment by dhirschfeld** [comment]
Thanks @manics for getting this one over the line - great work!

Hopefully someone from @conda-forge/core will be able to merge shortly...

**Comment by bollwyvl** [comment]
Thank you @beckermr !

**Pull request merged**